src/routing_table/prefix.rs

```diff
@@ -53,36 +53,36 @@ impl<T: Clone + Copy + Default + Binary + Xorable> Prefix<T> {
         if self.bit_count > 0 {
             self.bit_count -= 1;
             // unused bits should be zero:
-            self.name = self.name.with_bit(self.bit_count, false);
+            self.name = self.name.with_bit(self.bit_count(), false);
         }
         self
     }
 
     /// Returns the number of bits in the prefix.
     pub fn bit_count(&self) -> usize {
-        self.bit_count
+        self.bit_count as usize
     }
```

Comment:
Yeah - I agree, but I must admit that I never inline anything since it could just as easily cause a slowdown, and I think it can increase compile times.  I've been expecting that we'd look more closely at inlining when we one day get to the profiling stage.